Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseAutonomous.java

```diff
@@ -76,51 +91,124 @@ public void turnToAngle(double targetAngle) {
 
             // robot is turning counter-clockwise
             if (angleError > 0) {
-                motorPower = Math.min(angleError / -180.0, -0.2);
+                motorPower = Math.min(angleError / -250.0, -Constants.MINIMUM_TURNING_POWER);
```

Comment:
Why -250.0 here?

Reply:
That was to make sure the robot didn't turn too fast, now we have a constant for that

Reply:
The -250 reduces the power going to the motors